python/feathub/processors/local/local_processor.py

```diff
@@ -478,3 +485,121 @@ def _evaluate_python_udf_transform(
+        per_row_transform_features_following_first_sliding_feature = []
+
+        for feature in dependent_features:
+            if isinstance(feature.transform, ExpressionTransform):
+                if sliding_window_descriptor is not None:
+                    per_row_transform_features_following_first_sliding_feature.append(
+                        feature
+                    )
+                else:
+                    source_df[feature.name] = self._evaluate_expression_transform(
+                        source_df, feature.transform
+                    )
+            elif isinstance(feature.transform, PythonUdfTransform):
+                if sliding_window_descriptor is not None:
+                    per_row_transform_features_following_first_sliding_feature.append(
+                        feature
+                    )
+                else:
+                    source_df[feature.name] = self._evaluate_python_udf_transform(
+                        source_df, feature.transform
+                    )
+            elif isinstance(feature.transform, SlidingWindowTransform):
+                if feature_view.timestamp_field is None:
+                    raise FeathubException(
+                        "SlidingFeatureView must have timestamp field for "
+                        "SlidingWindowTransform."
+                    )
+                transform = feature.transform
+
+                if sliding_window_descriptor is None:
+                    sliding_window_descriptor = (
+                        SlidingWindowDescriptor.from_sliding_window_transform(transform)
+                    )
+
+                if (
+                    sliding_window_descriptor
+                    != SlidingWindowDescriptor.from_sliding_window_transform(transform)
+                ):
+                    raise FeathubException(
+                        "The SlidingWindowTransforms in a SlidingFeatureView should "
+                        "have the same step size and group by keys."
+                    )
+
+                agg_field_descriptors.append(
+                    AggregationFieldDescriptor.from_feature(feature)
+                )
+            else:
+                raise FeathubTransformationException(
+                    f"Unsupported transformation type "
+                    f"{type(feature.transform).__name__} for feature {feature.name}."
+                )
+
+        agg_df = evaluate_sliding_window(
+            input_df=source_df,
+            feature_view=feature_view,
+            window_descriptor=sliding_window_descriptor,
+            agg_descriptors=agg_field_descriptors,
+            tz=self.timezone,
+            parser=self.parser,
+            ast_evaluator=self.ast_evaluator,
+        )
+
+        for feature in per_row_transform_features_following_first_sliding_feature:
+            if isinstance(feature.transform, ExpressionTransform):
+                if feature.transform.expr == "CURRENT_EVENT_TIME()":
```

Comment:
Why do we need this `if` statement?

Reply:
The event time is computed in the `evaluate_sliding_window` method. This is a temporary solution to ignore the `CURRENT_EVENT_TIME` function. 

I added a TODO to properly handle the `CURRNET_EVENT_TIME` in the future.